coursebook/modules/m2/2-hands-on.ipynb

```diff
@@ -8,12 +8,14 @@
     "\n",
```

Comment:
<p>Are we expecting this to be done by parsing the category file with code, like in the prewritten code "code for later (skip for now!)"? Or will we prompt this to be more manual?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/rds-course/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
I was thinking probably just more manual for this one, `.replace(values_with_dict_I_made)` or something, but they could try parsing the category file if they want to.